### PR TITLE
docs: Update Theme article

### PR DESCRIPTION
**Pull request by NickIliev**
Related to [https://github.com/NativeScript/docs/issues/1846](https://github.com/NativeScript/docs/issues/1846)

DO NOT MERGE until full review and updated screenshots are pushed

### CODE REVIEW

File: docs/ui/theme.md [hunk 1]
```diff
@@ -18,56 +18,78 @@ The NativeScript project provides a core theme that you can add to any of your p
...
 ![Light and dark color schemes](/img/theme/color-schemes-light-and-dark.png)
 
-To use the light color scheme, add the following line of code to the top of your `app.css` file:
+> **Note:** The dark mode support was introduced natively with Android 10 and above and with iOS 13 and above.
+
+### CSS Imports
 
-``` CSS
-@import '~nativescript-theme-core/css/core.light.css';
+To use the default color scheme (with primary color blue), add the following lines of code to the top of your `app.css` file:
```
Comment (bundyo) [inline docs/ui/theme.md]:
```suggestion
To use the default color scheme (with accent color blue), add the following lines of code to the top of your `app.css` file:
```

File: docs/ui/theme.md [hunk 2]
```diff
@@ -18,56 +18,78 @@ The NativeScript project provides a core theme that you can add to any of your p
...
+@import '~@nativescript/theme/css/core.css';
+@import '~@nativescript/theme/css/default.css';
 ```
 
-And in order to use the dark scheme, add this line of code to your `app.css` instead:
+You may want to alternatively use one of the 11 other color schemes included in the NativeScript core theme. To do so, switch the **second** import in your `app.css` file to use one of the following imports.
 
+```CSS
+@import '~@nativescript/theme/css/aqua.css';
+@import '~@nativescript/theme/css/blue.css'; /* same as default.css */
```
Comment (bundyo) [inline docs/ui/theme.md]:
```suggestion
@import '~@nativescript/theme/css/blue.css';
```

Blue and Default skins are not the same - Default is the old Core skin.

File: docs/ui/theme.md [hunk 3]
```diff
@@ -18,56 +18,78 @@ The NativeScript project provides a core theme that you can add to any of your p
...
 ```
 
-> **TIP**: [This image](/img/theme/color-schemes-all.png) shows all 13 color schemes on iOS and Android, and can help you decide which color scheme is a good fit for your app.
+SCSS compatibility imports
+```CSS
+@import "~@nativescript/theme/core.compat";
+@import "~@nativescript/theme/blue.compat";
+```
+
+> **Note**: [This image](/img/theme/color-schemes-all.png) shows all 13 color schemes on iOS and Android, and can help you decide which color scheme is a good fit for your app.
```
Comment (bundyo) [inline docs/ui/theme.md:88]: I don't have access to this image?

File: docs/ui/theme.md [hunk 4]
```diff
@@ -171,25 +193,15 @@ Note that for each of these rules the “m” can be swapped with a “p” to a
...
 
 ``` XML
 <Label text="Separate this UI widget..."></Label>
-<StackLayout class="hr-dark m-10"></StackLayout>
+<StackLayout class="hr m-10"></StackLayout>
 <Label text="...from this UI widget."></Label>
 ```
 
-![dark dividers ios](/img/theme/dividers-dark-ios.png) ![dark dividers android](/img/theme/dividers-dark-android.png)
+![light dividers ios](/img/theme/dividers-light-ios.png)
```
Comment (bundyo) [inline docs/ui/theme.md:204]: Also can't access this image. Are you sure you commited it?

File: docs/ui/theme.md [hunk 5]
```diff
@@ -171,25 +193,15 @@ Note that for each of these rules the “m” can be swapped with a “p” to a
...
-``` XML
-<Label text="Separate this UI widget..."></Label>
-<StackLayout class="hr-light m-10"></StackLayout>
-<Label text="...from this UI widget."></Label>
-```
-
-![light dividers ios](/img/theme/dividers-light-ios.png) ![light dividers android](/img/theme/dividers-light-android.png)
-
-* `hr-dark`: Creates a dark colored divider.
+* `hr`: Creates a colored divider.
```
Comment (bundyo) [inline docs/ui/theme.md]: You might want to mention that the .hr class now adapts to the chosen mode.

File: docs/ui/theme.md [hunk 6]
```diff
@@ -230,223 +242,135 @@ You can convey meaning through color with a handful of utility classes that are
...
-  <StackLayout class="input-field">
-    <Label text="Option 2" class="label font-weight-bold m-b-5" />
-    <TextField class="input" />
-    <StackLayout class="hr-light"></StackLayout>
-  </StackLayout>
+<StackLayout class="nt-form">
+    <!-- Option 1: An input with no label, and a bottom border -->
+    <StackLayout class="nt-input">
+        <TextField hint="Option 1"/>
+        <StackLayout class="hr"></StackLayout>
```
Comment (bundyo) [inline docs/ui/theme.md]: This StackLayout shouldn't be needed - the TextField should have the bottom border already. Check here:
https://github.com/NativeScript/theme/blob/master/app/pages/forms.xml#L10

File: docs/ui/theme.md [hunk 7]
```diff
@@ -230,223 +242,135 @@ You can convey meaning through color with a handful of utility classes that are
...
-  <GridLayout class="input-field input-sides" rows="auto, auto" columns="*,*">
-    <Label text="Option 3" class="label font-weight-bold" row="0" col="0" />
-    <TextField class="input right" row="0" col="1" />
-    <StackLayout class="hr-light" row="1" colSpan="2"></StackLayout>
-  </GridLayout>
+    <!-- Option 2: An input with a label on top, and a bottom border -->
+    <StackLayout class="nt-input">
+        <Label text="Option 2" class="nt-label font-weight-bold m-b-5" />
+        <TextField/>
+        <StackLayout class="hr"></StackLayout>
```
Comment (bundyo) [inline docs/ui/theme.md]: Same here. Also the labels don't need nt-label class (which I think currently is not styled at all).

File: docs/ui/theme.md [hunk 8]
```diff
@@ -230,223 +242,135 @@ You can convey meaning through color with a handful of utility classes that are
...
+    </StackLayout>
 
-  <!-- Option 4: An input with a simple border and no label -->
-  <TextField hint="Option 4" class="input input-border" />
+    <!-- Option 3: An label and input—positioned side by side -->
+    <GridLayout class="nt-input -sides" rows="auto, auto" columns="*,*">
+        <Label text="Option 3" class="nt-label font-weight-bold" row="0" col="0" />
+        <TextField row="0" col="1"/>
+        <StackLayout class="hr" row="1" colSpan="2"></StackLayout>
```
Comment (bundyo) [inline docs/ui/theme.md:354]: Same here.
https://github.com/NativeScript/theme/blob/master/app/pages/forms.xml#L15

File: docs/ui/theme.md [hunk 9]
```diff
@@ -596,7 +467,7 @@ The first two sidedrawer class names control the alignment of content within the
 </StackLayout>
 ```
 
-* `sidedrawer-list-item`: A class name to apply to each individual item, or link, that appears within your sidedrawer’s content.
+* `sidedrawer-list-item`: A class name to apply to each item, or link, that appears within your sidedrawer’s content.
```
Comment (bundyo) [inline docs/ui/theme.md]: There are new classes here.

File: docs/ui/theme.md [hunk 10]
```diff
@@ -729,41 +628,160 @@ After that, paste the following code into your `app.android.scss` file.
...
+_app-common.scss_ 
+
+```CSS
+/* Colors */
+$accent: #369;
+
+/* This color was named $ab-background in Theme v1 */
+$complementary: fuchsia;
+
+/* The variables are loaded bofire the Core styles */
```
Comment (bundyo) [inline docs/ui/theme.md]:
```suggestion
/* The variables are loaded before the Core styles */
```

**Review by bundyo — CHANGES_REQUESTED** [review]
Some things need to be fixed, check comments.

File: docs/ui/theme.md (hunk 3 above)
Comment (NickIliev) [reply docs/ui/theme.md:88]: I' haven't touched this one - the image is present on my side.

File: docs/ui/theme.md (hunk 4 above)
Comment (NickIliev) [reply docs/ui/theme.md:204]: The image is committed and present in the local build.

File: docs/ui/theme.md (hunk 10 above)
Comment (NickIliev) [reply docs/ui/theme.md]: all changes applied

File: docs/ui/theme.md [hunk 11]
```diff
@@ -337,30 +337,26 @@ Here’s a form with several different form control display options you can expe
 ``` XML
 <StackLayout class="nt-form">
     <!-- Option 1: An input with no label, and a bottom border -->
-    <StackLayout class="nt-input">
-        <TextField hint="Option 1"/>
-        <StackLayout class="hr"></StackLayout>
-    </StackLayout>
+    <TextField hint="Option 1"/>
```
Comment (bundyo) [inline docs/ui/theme.md]: I didn't mean to remove the surrounding StackLayout, but the .hr one.

File: docs/ui/theme.md [hunk 12]
```diff
@@ -337,30 +337,26 @@ Here’s a form with several different form control display options you can expe
...
+    <StackLayout class="hr"></StackLayout>
 
     <!-- Option 2: An input with a label on top, and a bottom border -->
-    <StackLayout class="nt-input">
-        <Label text="Option 2" class="nt-label font-weight-bold m-b-5" />
-        <TextField/>
-        <StackLayout class="hr"></StackLayout>
-    </StackLayout>
+    <Label text="Option 2" class="font-weight-bold m-b-5" />
+    <TextField/>
```
Comment (bundyo) [inline docs/ui/theme.md]: Same here.

File: docs/ui/theme.md [hunk 13]
```diff
@@ -337,30 +337,26 @@ Here’s a form with several different form control display options you can expe
...
+    <StackLayout class="hr"></StackLayout>
 
     <!-- Option 3: An label and input—positioned side by side -->
-    <GridLayout class="nt-input -sides" rows="auto, auto" columns="*,*">
-        <Label text="Option 3" class="nt-label font-weight-bold" row="0" col="0" />
+    <GridLayout class="-sides" rows="auto, auto" columns="*,*">
```
Comment (bundyo) [inline docs/ui/theme.md]: Why remove the .nt-input class?

**Comment by bundyo** [comment]
Several more comments, the rest is fine.

**Review by bundyo — APPROVED** [review]

**Pull request merged**